test/test_helpers.py

```diff
@@ -168,6 +168,28 @@ def test_some_headers(self, patched_cache_folder):
         )
         assert examples.dataset.headers == ["im", ""]
 
+    def test_example_labels(self):
```

Comment:
I think this test is failing because you need `patched_cache_folder`?
```suggestion
    def test_example_labels(self, patched_cache_folder):
```